backend/algorithms/create.py

```diff
@@ -46,8 +49,9 @@
     CACHED_EXCLUSIONS = json.load(f)
 
 
-def create_category(tokens):
-    """Given a list of tokens starting from after the connector keyword, create
+def create_category(tokens) -> Category:
```

Comment:
this typehint is incorrect - we return a category and an int of how many tokens were used

Reply:
whoops!